api-client/src/endpoints/__tests__/createEventsEndpoint.spec.ts

```diff
@@ -1,5 +1,5 @@
-import { Moment } from 'moment'
-import moment from 'moment-timezone'
+// @ts-expect-error-module-has-no-exported-member
```

Comment:
@sarahsporck could you fix the problem with this error on import `DateTime`. It appeared after declaring module in types in api-client. It was done in order to get rid of the problem with null

Reply:
Basically we have to declaration merge the types form the library/definitely typed with our custom type setting `throwOnInvalid: true`.

Reply:
you do not correctly merge the luxon module, and because of that the original luxon types are lost. To fix this add `import 'luxon'` into the `luxon.d.ts` files, then the luxon namespace will be merged with the custom definitions.